R/utils.R

```diff
@@ -127,3 +127,24 @@ flatten_list <- function(x) {
         return(x)
     }
 }
+
+# This function gets called on package load.
+.onLoad <- function(libname, pkgname) {
+    # If credentials are in environment variables, use those.
+    creds <- aws.signature::locate_credentials()
+    if (!all(is.null(creds$key), is.null(creds$secret))) {
+        return
```

Comment:
I think this should be something other than just the `return` function. Do you mean `return(NULL)` or `return(invisible(NULL)` perhaps?